charters/MetadataSchema/charter.md

```diff
@@ -0,0 +1,66 @@
+
+# Metadata Schema
+
+## Description
+
+The Human Cell Atlas (HCA) Metadata Schema are JSON format schema. These schema are designed to capture and provide structure for the descriptive scientific metadata associated with HCA datasets. These schemas aim to ensure [the FAIRness of the HCA data](https://www.nature.com/articles/sdata201618).
+
+## Objectives
+
+1. Ensuring that HCA data are findable and the schemas are computationally queryable
+2. Ensuring that HCA metadata supports reproducibility and reusability
+3. Ensuring that HCA metadata has a consistent representation and can accurately represent the scientific description of a dataset in a computational readable manner
+4. Defining JSON schema which enable objectives 1 and 2
+5. Defining the semantic rules that the content follows
+6. Working with Ingest Infrastructure to specify the entity structure for HCA metadata schema
+7. Defining style guides and the release process for the schema
+8. Ensuring the defined metadata schema provides a positive user experience for those who read and write them
+
+## In-scope
+
+* Defining the specification for the schema and the schema itself
+* Defining the semantic rules for metadata content of the schema
+* Incorporating community feedback to extend and improve the schema
+* Ensuring the schema is compatible with open source software packages supporting the same draft of the JSON schema
+* Providing a test framework to enable community members to test if their proposed extensions are functional and meet our style guides
+* Providing exemplar data in the schema format
+* Providing computationally readable information about depreciated and moved fields within the schema
+* Working with data contributors to understand what types of descriptive fields they can supply for their samples and experiments
+* Working with data consumers to understand what descriptive fields they need in order to be able to analyse and visualise the HCA data
```

Comment:
It may be out of scope but I think it is worth calling out the importance of working with contributors/consumers to understand fields but also to understand usability. This may fall here but could also be incorporated more explicitly above as a part of incorporating community feedback.

Reply:
I believe working with contributors and consumers is very important and would be covered under objective 8:

"8. Ensuring the defined metadata schema provides a positive user experience for those who read and write them"

Reply:
@jonahcool are you happy this is covered by the objective

`Ensuring the defined metadata schema provides a positive user experience for those who read and write them`

and the inscope statements

```
Working with data contributors to understand what types of descriptive fields they can supply for their samples and experiments
Working with data consumers to understand what descriptive fields they need in order to be able to analyse and visualise the HCA data
Extending the schema to meet the needs of data contributors and data consumers based on the feedback they provide
```